README.md

```diff
@@ -46,6 +46,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [Git Secrets](#git-secrets)
 - [git-fixup](#git-fixup)
 - [git-recent](#git-recent)
+- [git-interactive-rebase-tool](#git-recent)
```

Comment:
oh the link is wrong. fixed in https://github.com/stevemao/awesome-git-addons/commit/6e9907b7bdcac38a5a6f08ee394ed1f169dad4b4